string marker_name
string new_tag